src/components/ProjectRequestForm/FormConifrmation.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export default function FormConifrmation({ formState, loadPreviousForm }) {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    alert(JSON.stringify(formState, null, 2));
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        please confirm form details:
+        {
+            Object.entries(formState).map(([key, value]) => {
+                return <div>{key}: {value}</div>
+            })
+        }
+        <button onClick={loadPreviousForm}>Previous</button>
+        <button type="submit">Submit</button>
+      </form>
+    </div>
```

Comment:
Is there a point to wrap a single form element in a div? You can return a form directly in your statement.

Reply:
You're right. Fixed